examples/integration_test.py

```diff
@@ -1202,9 +1202,117 @@ def verify_config(expconfig, configs):
     print("Topic {} marked for deletion".format(our_topic))
 
 
-# Exclude throttle since from default list
-default_modes = ['consumer', 'producer', 'avro', 'performance', 'admin']
+def verify_explicit_read():
```

Comment:
verify_avro_explicit_read_schema

call it from verify_avro()

Reply:
makes sense - c4ff376